weave-js/src/components/PagePanelComponents/Home/Browse3/typeViews/Audio/AudioPlayer.tsx

```diff
@@ -40,10 +53,11 @@ export const AudioPlayer: FC<{
   }
 
   const downloadFile = () => {
+    if (!audioUrl || !audioFileName) return;
```

Comment:
I wonder if we should error? either with a toast or even just to the browser console?

Reply:
I think a console error is fine, I removed the file name check since the url won't exist without the name.

Download button doesn't exist until button until the URL has loaded in and the player rendered so neither should ever occur.